admin/database/database.go

```diff
@@ -202,8 +213,8 @@ func (e *Variables) Scan(value interface{}) error {
 
 // InsertProjectOptions defines options for inserting a new Project.
 type InsertProjectOptions struct {
```

Comment:
Do we need `required` validation on other fields as well ? Like ProductionDriver and ProductionBranch ?

Reply:
It'll be rejected at the runtime level if not set correctly – maybe we'll supports driver autodetect in the future or something. Also didn't add `required` on all the foreign IDs since the DB constraints will check them.

Generally I just kept it light for this PR – but we might consider more rigorous/early validation later. Just wanted to get the basics in here.